lib/auth/auth.go

```diff
@@ -4255,7 +4256,7 @@ func (a *Server) addAdditionalTrustedKeysAtomic(
 func newKeySet(ctx context.Context, keyStore *keystore.Manager, caID types.CertAuthID) (types.CAKeySet, error) {
 	var keySet types.CAKeySet
 	switch caID.Type {
-	case types.UserCA, types.HostCA:
+	case types.UserCA, types.HostCA, types.OpenSSHCA:
```

Comment:
Does OpenSSH CA need TLS keys?

Reply:
Maybe not, I'm unsure tbh. I figured since it's essentially taking the place of the User CA when authenticating with registered OpenSSH nodes, it'd need what the User CA needs.

Reply:
I don't think we need the TLS cert. UserCA has TLS key because we're using it in more places (including the TLS handshake with the proxy). This CA will be only used to authenticate with OpenSSH. I think we should remove the TLS keys as we don't need them.

Reply:
Makes sense, done